documentation/doxygen/modifyClassWebpage.sh

```diff
@@ -15,7 +15,7 @@ libname=$(root -l -b -q "libs.C+g(\"$1\")" | grep 'mainlib=')
 
 # The class was not found. Remove the collaboration graph
 if [ -z "${libname}" ]; then
-   echo "WARNING modifyClassWebpage.sh: libs.C could not get library of class $1 from ROOT interpreter. Removing its collaboration diagram."
```

Comment:
I think this one is a real warning: sometimes ROOT is not able to find the library, even if there is one. It's related to this bug: https://github.com/root-project/root/issues/11667
Maybe changing it to "info" instead of "warning"?